src/microbe_stage/MicrobeStage.cs

```diff
@@ -244,6 +244,26 @@ public void MoveToEditor()
         parent.AddChild(editor);
     }
 
+    /// <summary>
+    ///   Switches to the main menu
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     TODO: This method should be in a singleton or something
```

Comment:
This could be put in GUICommon taking in the Current Node scene root so that it can detach it like here, right?

Reply:
Oh yeah, it could work.